packages/connect/src/base/listener.ts

```diff
@@ -417,20 +369,7 @@ class Listener extends EventEmitter implements InterfaceListener {
    * @returns list of addresses under which the node is available
    */
   getAddrs(): Multiaddr[] {
-    return (
-      [...this.addrs.external, ...this.addrs.relays, ...this.addrs.interface]
-        // Filter empty entries
-        .filter((addr) => addr)
-    )
-  }
-
-  /**
-   * Get amount of currently open connections
-   * @dev used for testing
-   * @returns amount of currently open connections
-   */
-  getConnections(): number {
-    return this.__connections.length
+    return [...this.addrs.external, ...this.entry.getUsedRelays(), ...this.addrs.interface]
```

Comment:
return ordered array since libp2p usually preserves that order, saves some unnecessary dials